pkg/chartverifier/certifier_test.go

```diff
@@ -111,3 +111,39 @@ func TestCertifier_Certify(t *testing.T) {
 
 	cancel()
 }
+
+func TestGetVersion(t *testing.T) {
+	t.Run("oc version error and wrong user input", func(t *testing.T) {
+		errIn := errors.New("error")
+		v, err := getVersion("", "NaN", errIn)
+		require.NoError(t, err)
+		require.Equal(t, "NaN", v)
+	})
```

Comment:
Assuming the function gets inlined as reviewed, those tests could be moved to the verify command tests instead, and exercized with the equivalent command line argument.

Reply:
I don't object to the function and the more tests we have the better, with the function we can have the both sets of tests.

Would rather the function also had the logic to set  OpenShiftVersionErr and OpenShiftSemVerErr to encapsulate that functionality better, and get more from the test but not enough to hold up the fix.

LGTM

Reply:
Would like to see the test cover all scenarios, not just one.